### PR TITLE
Add support for AccessScopes

### PR DESCRIPTION
**Pull request by sshaw**

### CODE REVIEW

File: access_scopes.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+type AccessScopeService interface {
+	List(options interface{}) ([]AccessScope, error)
+}
+
+type AccessScopeServiceOp struct {
+	client *Client
+}
+
+func (a *AccessScopeServiceOp) List(options interface{}) ([]AccessScope, error) {
```
Comment (sshaw) [inline access_scopes.go:19]: This [takes no arguments](https://shopify.dev/docs/admin-api/rest/reference/access/accessscope#index-2021-04) but there are other cases like this and `options interface{}` is still used so, I added it.

**Comment by sshaw** [comment]
Anybody home‽

**Comment by BoldBrandonM** [comment]
Hey @sshaw, apologies that we did not get to this sooner. Internally our competing priorities have seen us unavailable to respond here. Rather than disturb you with your change from ~9 months ago, we opted to go with a version of the change to get it available ASAP.

Your PR here served as a fantastic guide and we thank you for the contribution to go-shopify. For your reference, the PR we ended up going with was this one (which is available in the new 3.12.0 tag):
https://github.com/bold-commerce/go-shopify/pull/180

Hope you're well, stay safe out there!

**Comment by BoldBrandonM** [comment]
Hey @sshaw, apologies that we did not get to this sooner. Internally our competing priorities have seen us unavailable to respond here. Rather than disturb you with your change from ~9 months ago, we opted to go with a version of the change to get it available ASAP.

Your PR here served as a fantastic guide and we thank you for the contribution to go-shopify. For your reference, the PR we ended up going with was this one (which is available in the new 3.12.0 tag):
https://github.com/bold-commerce/go-shopify/pull/180

Hope you're well, stay safe out there!

**Comment by sshaw** [comment]
Hi, thanks for the update. Glad to see the feature is available. Hopefully you will be able to checkout the other PRs as there is a lot of pending functionality!

**Pull request closed without merging**